package.json

```diff
@@ -82,10 +81,11 @@
     "@babel/preset-env": "7.1.0",
     "@babel/preset-react": "7.0.0",
     "@babel/runtime": "7.1.2",
-    "@hot-loader/react-dom": "^16.7.0",
-    "@wordpress/babel-preset-default": "^4.11.0",
-    "@wordpress/components": "8.5.0",
-    "@wordpress/element": "2.10.0",
+    "@hot-loader/react-dom": "^16.14.0",
+    "@wordpress/babel-preset-default": "^5.0.1",
+    "@wordpress/base-styles": "^3.3.0",
+    "@wordpress/components": "^12.0.7",
```

Comment:
This is a major update of @wordpress/components from 8.5 to 12.0.7

Reply:
I went back to 11.1.15 because the dependencies are messy on 12 and I can't fix it so far